flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/factories/TestValuesTableFactory.java

```diff
@@ -457,6 +457,22 @@ private static RowKind parseRowKind(String rowKindShortString) {
                     .withDescription(
                             "Option to determine whether or not to require the distribution bucket count");
 
+    private static final ConfigOption<Boolean> SINK_SUPPORTS_DELETE_ON_KEY =
+            ConfigOptions.key("sink.supports-delete-on-key")
+                    .booleanType()
+                    .defaultValue(false)
+                    .withDescription(
+                            "Option to determine whether or not to require deletes to have the"
```

Comment:
nit: I wonder if we can simplify the words WDYT?
`Option to determine whether or not to require deletes to have the entire row or is a delete by key sufficient.`
->
`Indicates to deletes will be done by key; otherwise deletes will be done by record.`